test/clients/azure/osa.go

```diff
@@ -44,13 +43,12 @@ func (cli *Client) UpdateOSACluster(ctx context.Context, external *v20180930prev
 	// create logger
 	logrus.SetLevel(log.SanitizeLogLevel("Debug"))
 	logrus.SetFormatter(&logrus.TextFormatter{FullTimestamp: true})
-	logger := logrus.NewEntry(logrus.StandardLogger())
 
 	var oc *v20180930preview.OpenShiftManagedCluster
 	var err error
 	// simulate the API call to the RP
-	if oc, err = fakerp.CreateOrUpdate(ctx, external, logger, config); err != nil {
-		return nil, err
-	}
-	return oc, nil
+	//if oc, err = fakerp.CreateOrUpdate(ctx, external, logger, config); err != nil {
```

Comment:
why is this commented out?  is a middle ground necessary before this entire function gets deleted?

Reply:
Waiting on https://github.com/openshift/openshift-azure/pull/880 to merge, then I am going to fix this.